tests/tier1/tc_1111_template_conf_validation.py

```diff
@@ -0,0 +1,46 @@
+# coding:utf-8
+from virt_who import *
+from virt_who.base import Base
+from virt_who.register import Register
+from virt_who.testing import Testing
+
+
+class Testcase(Testing):
+    def test_run(self):
+        self.vw_case_info(os.path.basename(__file__), case_id='RHEL-195889')
+        trigger_type = self.get_config('trigger_type')
+        hypervisor_type = self.get_config('hypervisor_type')
+        if trigger_type != 'trigger-rhel':
+            self.vw_case_skip(trigger_type)
+        if hypervisor_type != 'esx':
+            self.vw_case_skip(hypervisor_type)
+
+        results = dict()
+        options = [
+                '#[config name]',
+                '#type=',
+                '#server=',
+                '#username=',
+                '#password=',
+                '#encrypted_password=',
+                '#owner=',
+                '#hypervisor_id=',
+                '#rhsm_hostname=',
+                '#rhsm_port=',
+                '#rhsm_username=',
+                '#rhsm_password=',
+                '#rhsm_encrypted_password=',
+                '#rhsm_prefix=/rhsm',
+                '#kubeconfig=',
+                '#kubeversion=',]    
+        self.pkg_uninstall(self.ssh_host(), 'virt-who')
+        self.pkg_install(self.ssh_host(), 'virt-who')
+        _, output = self.runcmd('cat /etc/virt-who.d/template.conf', self.ssh_host())
```

Comment:
Seems some caese will use the `vw_etc_d_delete_all` function to delete the files in the `/etc/virt-who.d/` (such as tc_1068,tc_1069,tc_1070,tc_1071), will this have an impact on this case?

Reply:
So we should call `pkg_uninstall()` and `pkg_install()` to re-install virt-who package for this case, just make sure /etc/virt-who.d/template.conf is recovered for this case.

Reply:
ok :)